Source/WebCore/rendering/RenderBoxModelObject.cpp

```diff
@@ -321,14 +321,24 @@ DecodingMode RenderBoxModelObject::decodingModeForImageDraw(const Image& image,
     if (!bitmapImage)
         return DecodingMode::Synchronous;
 
+    auto isUserScrollInProgress = [&]() -> bool {
+        RenderLayer* enclosingClippingLayer = hasLayer() ? layer()->enclosingOverflowClipLayer(ExcludeSelf) : nullptr;
+        if (!enclosingClippingLayer)
+            return false;
+
+        if (auto* scrollableArea = enclosingClippingLayer->scrollableArea())
+            return scrollableArea->isUserScrollInProgress();
```

Comment:
I'm not sure that "being in a user scroll" is the best criterion. What about animated programmatic scrolls? What about CSS transform animations?

Reply:
This was removed.